src/functionalTest/resources/features/common/ts_endpoints/Put_Dictionary_Base_Definition_Store.td.json

```diff
@@ -0,0 +1,9 @@
+{
+  "_guid_": "Put_Dictionary_Base_Definition_Store",
```

Comment:
Nothing is currently using this, it look like

S-004.8_Existing_Data.td.json
S-004.8.td.json
S-004.9.td.json

should all be extending from it (and therefore no need for the `"s2sClientId": "ccd_definition",` line in the above files)

Reply:
I forgot to remove this file so it is now remove